teerex/src/weights.rs

```diff
@@ -67,13 +67,11 @@ impl<T: frame_system::Config> WeightInfo for IntegriteeWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(2))
 			.saturating_add(T::DbWeight::get().writes(1))
 	}
-	// TODO benchmark dcap registration
```

Comment:
Oups, accidentally mistake from cherry-pick, but the next PR will resolve it anyhow.